content/docs/start/data-and-model-versioning.md

```diff
@@ -109,10 +109,9 @@ $ git commit -m "Configure remote storage"
 
 DVC remotes let you store a copy of the data tracked by DVC outside of the local
 cache (usually a cloud storage service). For simplicity, let's set up a _local
-remote_:
+remote_ in `/tmp/dvcstore` (create the dir first if needed):
```

Comment:
I think this can be replaced by  `$TMPDIR/dvcstore` even in Linux. For certain configurations `/tmp` may not exist or not configured the same as Ubuntu. (e.g., Android, or Docker.)

Reply:
`$TMPDIR` may be missing in Linux, so there seems to be no simple way to determine the temporary directory in an OS-independent fashion. Better to keep this way.